model-engine/model_engine_server/core/loggers.py

```diff
@@ -123,10 +119,14 @@ def make_json_logger(name: str, log_level: int = logging.INFO) -> logging.Logger
     in_kubernetes = os.getenv("KUBERNETES_SERVICE_HOST")
     if in_kubernetes:
         stream_handler.setFormatter(CustomJSONFormatter())
+    else:
+        stream_handler.setFormatter(CustomJSONFormatter())
```

Comment:
is this for testing? what about just remove if

Reply:
yeah this was for testing and can be removed, I think that we want to leave the logging as basic logging for testing locally with `just up` so that we don't have to read through the JSON logs.